app/assets/javascripts/code/services/_fixed_sessions.js

```diff
@@ -129,7 +129,7 @@ export const fixedSessions = (
 
     drawSessionsInLocation: function() {
       map.markers = [];
-      _(this.get()).each(session => drawSession.drawFixedSession(session, boundsCalculator(this.sessions)));
+      _(this.get()).each(session => drawSession.drawFixedSession(session));
```

Comment:
If possible it would be great if we could reduce the usage of `_` so we can move to something like ramda. Example
```
(this.get() || []).forEach(drawSession.drawFixedSession)
```
But prolly `this.get()` should always return an array (ie `(this.get() || [])` not needed)?

Reply:
`[]` is not needed

Reply:
I think that was done case `null`, `undefined` or a single value become array with `_()`. Example `_(1) == [1]` or `_(null) == []`